### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @CoJaques

### DISCUSSION THREAD

**Discussion by CoJaques** [heig-vd-dai-course/heig-vd-dai-course#46] [Show and tell]
https://github.com/CoJaques/DAI-Maven-Project.git

**Comment by ludelafo** [comment]
Thank you!

I would use code blocks (with ```` ```sh ```` for Shell scripts) to improve the reading of the commands. Also, I would add the commands to build the application from the command line if someone can't/doesn't want to run IntelliJ to run your application.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
